src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -160,6 +160,15 @@ class ChatSwitcherView extends React.Component {
             .map((report) => {
                 const participants = lodashGet(report, 'participants', []);
                 const isSingleUserDM = participants.length === 1;
+                const personalDetails = this.props.personalDetails
```

Comment:
I think using the `||` here would be more readable, and I think it's what we normally use.
``` javascript
const personalDetails = this.props.personalDetails || {};
```

Reply:
Alright. Fix coming up in my next commit.